src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -261,8 +274,12 @@ public void onTraversalStart() {
         intervals = hasUserSuppliedIntervals() ? intervalArgumentCollection.getIntervals(getBestAvailableSequenceDictionary()) :
                 Collections.emptyList();
 
-        Collection<Annotation>  variantAnnotations = makeVariantAnnotations();
-        annotationEngine = new VariantAnnotatorEngine(variantAnnotations, dbsnp.dbsnp, Collections.emptyList(), false, keepCombined);
+        final Collection<Annotation>  variantAnnotations = makeVariantAnnotations();
+        final GATKAnnotationPluginDescriptor pluginDescriptor = getCommandLineParser().getPluginDescriptor(GATKAnnotationPluginDescriptor.class);
+        final List<String> annotationStringsToKeep = pluginDescriptor.getUserArgs().getKeepSpecifiedCombinedAnnotationNames();
+        final Map<String, Annotation> allDiscoveredAnnotations = pluginDescriptor.getAllDiscoveredAnnotations();
+        final Set<Annotation> annotationsToKeep = annotationStringsToKeep.stream().map(allDiscoveredAnnotations::get).collect(Collectors.toSet());
```

Comment:
I think it would be preferable to add an override of `makeVariantAnnotations` into this class (adjacent to the `getPluginDescriptors` override), and move the computation of which annotations to keep into that, rather than inlining it here.

Also, you're basically giving priority to the new arg in the case where it conflicts with, say, an exclude arg, which I think is fine if thats what you want, but you might want to document that as part of the new arg if thats what you go with.

Reply:
The problem is I still need the regular list of annotations (in addition to the `annotationsToKeep`). I can still move it to a separate function though. 

Also I think I resolved this by not using `allDiscoveredAnnotations` so now excluded args should get handled correctly.